test/sagemaker_tests/mxnet/training/integration/local/test_horovod.py

```diff
@@ -18,8 +18,7 @@
 
 import pytest
 from sagemaker.mxnet import MXNet
-from test.integration import RESOURCE_PATH
-
+from ...integration import RESOURCE_PATH
```

Comment:
test_horovod is recently added and it expects additional arguments while running the pytest commond. This wont be fixed by import change in the pr